cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -50,6 +50,11 @@ function up() {
             $kubectl create -f /opt/cnao/network-addons-config-example.cr.yaml
             $kubectl wait networkaddonsconfig cluster --for condition=Available --timeout=200s
         fi
+
+        # When CNAO is enabled, start the CNI dhcp daemon on every node by default
+        for node in $( seq -w 1 99 | head -n $KUBEVIRT_NUM_NODES ); do
+            ${_cli} --prefix $provider_prefix ssh node$node -- sudo systemctl enable --now cni-dhcp
+        done
```

Comment:
Is there an option to do this from the tests? It would help support running the U/S tests on any D/S (not only kubevirtci). Conformance tests could also take advantage of this.

Reply:
Sorry I missed that one!
Starting and stopping services on every node from the test suite feels a bit cumbersome to me.
I guess my question would be: is there ever a reason to *not* be running this daemon? We (and everybody else) could just enable it by default and be done with this.

Reply:
Yes, I agree, it could be enabled without condition. The problem is, that it may not be started in practice on other clusters. I guess it will be enough to make sure the test is not ran on clusters where the service is not running.

I think the main point of all this is that we introduce a new dependency on the cluster, for running the tests. If we could avoid it, it would have been simpler for all.
(e.g. using static IP)